java/src/jmri/jmrit/beantable/RowComboBoxPanel.java

```diff
@@ -86,15 +84,15 @@
      *  @param customRenderer renderer to display things
      */
     public RowComboBoxPanel(Object [] values,
-                            ListCellRenderer customRenderer) {
-        super (new JComboBox());
+                            ListCellRenderer<?> customRenderer) {
+        super (new JComboBox<>());
         // is being filled from HashMap
         this.editor = new JPanel(new BorderLayout ());
         if (values != null) {
             setItems(values); // in 4.5.7 this is not yet called using values, but might be useful in a more general application
         }
         this.renderer = new JPanel(new BorderLayout ());
-        setClickCountToStart(1); // value for a DefaultCellEditor: immediately start editing
+        super.setClickCountToStart(1); // value for a DefaultCellEditor: immediately start editing
```

Comment:
Is this invoking of `super.setClickCountToStart(1)` a functional change, where it was calling the `this. super.setClickCountToStart(1)` method before? Or was it needed for some other reason?

Reply:
This was flagged as a "Overrideable method call in constructor" warning, no functional change intended.